app/helpers/automatic/daily_room_pool.py

```
@@ -0,0 +1,487 @@
+
+            room.is_used = True
+            self.active_rooms[session_id] = room
+
+            # If the pool is low, start creating a new room in the background.
+            if (
+                self.available_rooms.qsize() <= 1
+                and not self.is_creating_room
+                and self._current_total_rooms() < self.max_pool_size
+            ):
+                logger.info("Room pool getting low, creating background room")
+                asyncio.create_task(self._create_background_room())
```

Ruff (0.13.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)